app/controllers/application_controller.rb

```diff
@@ -1,3 +1,2 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```

Comment:
We will need this. I expect this app to serve an admin interface so I don't want to strip out CSRF protection.

Reply:
Fixed in 0f3bccfde3ad3757193cc0710d54a1d1ac0c8320